android/src/main/kotlin/com/mapbox/maps/mapbox_maps/MapboxMapController.kt

```diff
@@ -181,6 +183,10 @@ class MapboxMapController(
         gestureController.removeListeners()
         result.success(null)
       }
+      "platform#releaseMethodChannels" -> {
+        releaseMethodChannels()
```

Comment:
It would just be necessary to change the name releaseMethodChannels to a more explicit name